pkg/sentry/devices/nvproxy/version.go

```diff
@@ -502,6 +504,7 @@ func Init() {
 							nvgpu.NV2080_CTRL_CMD_RC_RELEASE_WATCHDOG_REQUESTS:                     nil, // No params.
 							nvgpu.NV2080_CTRL_CMD_RC_SOFT_DISABLE_WATCHDOG:                         nil, // No params.
 							nvgpu.NV2080_CTRL_CMD_TIMER_GET_GPU_CPU_TIME_CORRELATION_INFO:          simpleIoctl("NV2080_CTRL_TIMER_GET_GPU_CPU_TIME_CORRELATION_INFO_PARAMS"),
+							nvgpu.NV2080_CTRL_CMD_TIMER_SET_GR_TICK_FREQ:                           simpleIoctl("NV2080_CTRL_CMD_TIMER_SET_GR_TICK_FREQ"),
```

Comment:
`NV2080_CTRL_CMD_TIMER_SET_GR_TICK_FREQ_PARAMS`. This is for mapping what driver struct is mapped to this control command.

Reply:
oops, thank you!